### ISSUE TITLE
Is RSQLParser.parse() thread-safe?

### ISSUE THREAD

**Issue by gevaerts** [jirutka/rsql-parser#34]
It would be useful if the documentation says whether parsing is thread-safe.

Based on a quick look at what RSQLParser.parse() does, I think it's thread-safe. Is that correct?

**Comment by jmzc** [comment]
@gevaerts , I agree with you: looking at source code , I think RSQLParser is thread-safe.
But it would be nice if someone else could confirm it

**Comment by qnhant5010** [comment]
Several prod tests agree that `RSQLParser` is thread-safe.
